test/e2e/network_segmentation.go

```diff
@@ -62,314 +62,320 @@ var _ = Describe("Network Segmentation", func() {
 			workerTwoNodeName            = "ovn-worker2"
 		)
 
-		DescribeTable(
-			"can perform east/west traffic between nodes",
-			func(
-				netConfigParams networkAttachmentConfigParams,
-				clientPodConfig podConfiguration,
-				serverPodConfig podConfiguration,
-			) {
-				netConfig := newNetworkAttachmentConfig(netConfigParams)
-
-				netConfig.namespace = f.Namespace.Name
-				clientPodConfig.namespace = f.Namespace.Name
-				serverPodConfig.namespace = f.Namespace.Name
-
-				By("creating the attachment configuration")
-				_, err := nadClient.NetworkAttachmentDefinitions(f.Namespace.Name).Create(
-					context.Background(),
-					generateNAD(netConfig),
-					metav1.CreateOptions{},
+		DescribeTableSubtree("create network using",
```

Comment:
more readable suggestion: "created using", then options "NAD" and "UDN"

Reply:
Done

Reply:
I think you lost one letter :D it says now "create using"

Reply:
Fixed, thanks